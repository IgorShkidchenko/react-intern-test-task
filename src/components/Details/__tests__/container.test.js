import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import DetailsContainer from "../container";

configure({ adapter: new Adapter() });

describe("<DetailsContainer />", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<DetailsContainer location={{ state: { repository: {} } }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
