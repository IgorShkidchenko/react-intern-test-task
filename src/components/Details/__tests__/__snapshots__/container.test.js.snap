// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DetailsContainer /> renders correctly 1`] = `
<DetailsComponent
  repository={Object {}}
/>
`;
